### PR TITLE
Add channels last, ResNet, and ResNext sizes for Batch Norm benchmarks.

### PR DESCRIPTION
**Pull request by csarofeen**

### CODE REVIEW

File: torch/csrc/jit/codegen/cuda/ir_interface_nodes.h [hunk 1]
```diff
@@ -469,7 +469,7 @@ class TORCH_CUDA_CU_API TensorView : public Val {
   void setDomain(TensorDomain* td) {
     domain_ = td;
   }
-
+public:
```
Comment (naoyam) [inline torch/csrc/jit/codegen/cuda/ir_interface_nodes.h]: Is this intentional?

Comment (naoyam) [reply torch/csrc/jit/codegen/cuda/ir_interface_nodes.h]: Oh, was this copied from my tuning change?

Comment (csarofeen) [reply torch/csrc/jit/codegen/cuda/ir_interface_nodes.h]: Oh yeah, let me revert that, was testing with it, didn't mean to grab this change.

**Review by naoyam — APPROVED** [review]
LGTM.

File: benchmarks/cpp/nvfuser/batch_norm_nhwc.cpp [hunk 2]
```diff
@@ -0,0 +1,353 @@
...
+        at_x,
+        ato_weight,
+        ato_bias,
+        ato_run_mean,
+        ato_run_var,
+        true,
+        kMomentum,
+        kEps,
+        true);
+    // auto output = at::_batch_norm_impl_index(
```
Comment (rdspring1) [inline benchmarks/cpp/nvfuser/batch_norm_nhwc.cpp]: Is this needed?

File: benchmarks/cpp/nvfuser/CMakeLists.txt [hunk 3]
```diff
@@ -2,6 +2,8 @@ if(USE_CUDA)
   add_executable(nvfuser_bench
     batch_norm.cpp
     batch_norm_backward.cpp
+    batch_norm_nhwc.cpp
```
Comment (rdspring1) [inline benchmarks/cpp/nvfuser/CMakeLists.txt]: Nit: I think renaming the files from `batch_norm_nhwc` to `batch_norm_channels_last` makes things clearer.
Ditto: from `batch_norm` to `batch_norm_contiguous` or `batch_norm_channels_first`

**Review by rdspring1 — APPROVED** [review]
LGTM.

**Comment by rdspring1** [comment]
I think `batch_norm.cpp` and `batch_norm_nhwc.cpp` share a lot of code. Would it make sense to merge them later?

**Comment by csarofeen** [comment]
Yeah, it probably would make sense to try and unify them. Though I don't think the duplication is too bad. If we had a 3rd instance of batch norm benchmarks I'd probably say we should do it now, but with just 2 I'm not too concerned.
